atmos.yaml

```
@@ -91,6 +91,44 @@ logs:
   # Can also be set using 'ATMOS_LOGS_LEVEL' ENV var, or '--logs-level' command-line argument
   level: Info
 
+validate:
+  # The configuration settings for the editorconfig-checker.
+
+  editorconfig:
+    # A comma-separated list of file paths or patterns to exclude from checks.
+    exclude: ""
+
+    # If set to true, the default ignore patterns (like .git/*) will not be applied.
+    ignore_defaults: false
+
+    # Runs the checker without making any changes or producing output, useful for testing configuration.
+    dry_run: false
+
+    # Specifies the output format. Options: "default", "json".
+    format: "default"
+
+    # Disables colored output in the terminal.
+    no_color: false
```

Let's use atmos conventions of positive booleans, so we don't use double negatives.

```suggestion
    # Toggles colored output in the terminal.
    color: true
```

---

Done.